web/src/message_view_header.js

```diff
@@ -199,9 +174,7 @@ export function open_search_bar_and_close_narrow_description() {
 }
 
 export function close_search_bar_and_open_narrow_description() {
-    const filter = narrow_state.filter();
```

Comment:
A lot of callsites were doing this filter check anyways, so I consistently had them do it before calling this function so that the function always does something as its name suggests. (I can also make this a prep commit.)